app/api/sponsors.py

```diff
@@ -109,7 +111,8 @@ class SponsorDetail(ResourceDetail):
     """
     Sponsor detail by id
     """
-    decorators = (jwt_required,)
+    decorators = (api.has_permission('is_coorganizer', methods="PATCH,DELETE",
```

Comment:
@poush Weird . Mine works without it. It is implemented in same way in microlocations too. Can you confirm ?